firmware/quadruna/VC/src/app/states/app_initState.c

```diff
@@ -40,27 +42,28 @@ static void initStateRunOnEntry(void)
     io_efuse_setChannel(EFUSE_CHANNEL_BUZZER, false);
 }
 
-static void initStateRunOnTick100Hz(void)
-{
-    // Turn on inverters when requested to do so by the BMS (so we can power them up before HV is applied).
-    // Also, turn them on if HV is already up/coming up.
-    const bool enable_inverters = app_canRx_BMS_State_get() == BMS_INVERTER_ON_STATE ||
-                                  app_canRx_BMS_State_get() == BMS_PRECHARGE_STATE ||
-                                  app_canRx_BMS_State_get() == BMS_DRIVE_STATE;
-    if (enable_inverters)
-    {
-        app_stateMachine_setNextState(app_inverterOnState_get());
-    }
-}
-
 static void initStateRunOnTick1Hz(void)
 {
     app_allStates_runOnTick1Hz();
 
+    // TODO: will be changing this, putting it in HV init
     app_pumpControl_setFlowRate(
         app_canRx_Debug_SetCoolantPump_CustomEnable_get() ? app_canRx_Debug_SetCoolantPump_CustomVal_get() : 600.0f);
```

Comment:
should I put this in `HV Init` state now? Or for the next car with the new pump system?